### PR TITLE
fix:  limit not being updated on members deletion

### PR DESCRIPTION
**Pull request by ArmanNik**
Also fixed page invalidation

### CODE REVIEW

File: src/lib/constants.ts [hunk 1]
```diff
@@ -8,6 +8,7 @@ export enum Dependencies {
     ADDRESS = 'dependency:address',
     PAYMENT_METHODS = 'dependency:paymentMethods',
     ORGANIZATION = 'dependency:organization',
+    MEMBERS = 'dependency:members',
```
Comment (stnguyen90) [inline src/lib/constants.ts:11]: Do we need to make sure to invalidate this when adding a member too?

File: src/routes/console/organization-[organization]/deleteMember.svelte [hunk 2]
```diff
@@ -25,6 +29,11 @@
             } else {
                 dispatch('deleted');
             }
+            invalidate(Dependencies.ACCOUNT);
```
Comment (stnguyen90) [inline src/routes/console/organization-[organization]/deleteMember.svelte:32]: Why do we need to invalidate account?

Comment (ArmanNik) [reply src/routes/console/organization-[organization]/deleteMember.svelte:32]: Not sure but it was there and I'm afraid to remove it :rofl:

File: src/lib/constants.ts (hunk 1 above)
Comment (ArmanNik) [reply src/lib/constants.ts:11]: Fixed :)

**Review by stnguyen90 — APPROVED** [review]

**Review by TorstenDittmann — APPROVED** [review]

**Pull request merged**